### ISSUE TITLE
Why Very Big Spike in Learning Curve

### ISSUE THREAD

**Issue by Localcrack** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1743]
As you see in the below image the curve is almost ok till 10K steps then it again goes to 9K steps backward and then again goes forward.
why this happened ? is something wrong ?

![Screenshot 2024-01-19 152016](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/139331483/4349cd84-634f-4f6b-9960-adcff36bafe1)

This spike is shows in all graphs.

![Screenshot 2024-01-19 152102](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/139331483/2cbea2ce-0f24-4a17-83f8-b2a76e4aa1ec)

**Comment by arbaz686** [comment]
Check your learning rate schedule. If you are using a constant learning rate, try decreasing it. Also, implement a learning rate decay if it's not already being used.

If you are resuming training from a checkpoint, ensure that the optimizer state is also being restored correctly.
